flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -231,6 +231,31 @@ public class YarnConfigOptions {
                     .withDescription(
                             "A comma-separated list of tags to apply to the Flink YARN application.");
 
+    /**
+     * Users and groups to give VIEW access.
+     * https://www.cloudera.com/documentation/enterprise/latest/topics/cm_mc_yarn_acl.html
+     */
+    public static final ConfigOption<String> APPLICATION_VIEW_ACLS =
+            key("yarn.view.acls")
+                    .stringType()
+                    .defaultValue("")
```

Comment:
I think we could use the `.noDefaultValue()` here

Reply:
Fixed.